sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/models/package-info.java

```diff
@@ -2,5 +2,5 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-/** Package containing the data models for CommunicationIdentityClient. Azure Communication Identity Service. */
+/** Package containing classes for CommunicationIdentityClient. Azure Communication Identity Service. */
```

Comment:
Remove references of CommunicationIdentityClient

Reply:
Updated